build.gradle

```diff
@@ -1,38 +1,33 @@
 buildscript {
 	repositories {
+		// External libs - jcenter is Bintray and is supposed to be a superset of Maven Central, but do both just in case
+		jcenter()
+		mavenCentral()
 		maven {
-			url "https://plugins.gradle.org/m2/"
+			url = uri("https://plugins.gradle.org/m2/")
 		}
 	}
+
 	dependencies {
-		classpath "gradle.plugin.net.minecrell:licenser:0.3"
+		//Spotbugs
+		classpath("gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.0.0")
+
+		// SonarQube / Cloud scanning
+		classpath("org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:2.8")
 	}
 }
 
-apply plugin: 'java'
-apply plugin: 'idea'
-apply plugin: 'net.minecrell.licenser'
-
-repositories {
-	mavenCentral()
-	maven {
-	url "http://artifactory.terasology.org/artifactory/virtual-repo-live"
-	}
+plugins {
+	id("terasology-module")
 }
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
 
 dependencies {
 	compile group: 'com.google.code.gson', name: 'gson', version: '2.8.5'
 	compile group: 'org.msgpack', name: 'msgpack-core', version: '0.8.16'
 	compile fileTree(dir: 'libs', include: '*.jar')
-	testCompile 'junit:junit:4.12'
+
 	compile group: 'org.terasology.jnlua', name: 'JNLua', version: '0.1.0-SNAPSHOT'
 }
 
-license {
-	sourceSets = [project.sourceSets.main]
-	header = project.file("docs/NOTICE")
-	ignoreFailures = true
-}
```

Comment:
Should we keep this? what exactly does it do? Removing some license mangling sounds weird...

Reply:
There are indeed some special license considerations for this when in the context of the standalone Kallisti project, not the Terasology module :-)